src/table/header-cell/th-element.tsx

```diff
@@ -58,7 +71,13 @@ export function TableThElement({
     <th
       data-focus-id={`header-${String(columnId)}`}
       className={clsx(
-        className,
+        styles['header-cell'],
+        styles[`header-cell-variant-${variant}`],
+        sticky && styles['header-cell-sticky'],
+        stuck && styles['header-cell-stuck'],
```

Comment:
Here, "sticky" means the header has the sticky feature and "stuck" means the page is scrolled enough so that the sticky header starts covering part of the table.